homeassistant/components/media_player/kodi.py

```diff
@@ -679,6 +691,33 @@ def async_set_shuffle(self, shuffle):
             {"playerid": self._players[0]['playerid'], "shuffle": shuffle})
 
     @asyncio.coroutine
+    def async_run_method(self, method, **kwargs):
+        """Run Kodi JSONRPC API method with params."""
+        import jsonrpc_base
+        _LOGGER.debug('Run API method "%s", kwargs=%s', method, kwargs)
+        result_ok = False
+        try:
+            if kwargs:
+                result = yield from getattr(self.server, method)(kwargs)
```

Comment:
It works both ways, now that I check it, I'll leave it simpler, without the if condition.

And I'll change the names.